docs/TUTORIAL.md

```diff
@@ -287,33 +325,40 @@ Note that fogg works by generating Terraform and Make files. It does not run any
     .
     ├── Makefile
     ├── README.md
-    ├── fogg.json
+    ├── fogg.yml
+    ├── fogg.yml~
```

Comment:
what's this second file about actually 😮

Reply:
It's just a backup file that is automatically generated. It's harmless but not sure if it needs to be deleted or not. Thoughts @ryanking

Reply:
That is probably created by your editor, not fogg, so you should probably leave it out of the listing.